File: 23_C#Basics/_04_LoopsPartOneMoreExercises/_10_Clock2/_10_Clock2.cs
using System;

public class _10_Clock2
{
    public static void Main(string[] args)
    {
        // Print time
        for (int i = 0; i < 24; i++)
        {
            for (int j = 0; j < 60; j++)
            {
                for (int k = 0; k < 60; k++)
                {
                    Console.WriteLine($"{i} : {j} : {k}");
                }
            }
        }
    }
}

File: 01_ProgrammingBasics/_04_LoopsPartOneMoreExercises/_10_Clock2.java
package _01_ProgrammingBasics._04_LoopsPartOneMoreExercises;

public class _10_Clock2 {
    public static void main(String[] args) {
        // Print time
        for (int i = 0; i < 24; i++) {
            for (int j = 0; j < 60; j++) {
                for (int k = 0; k < 60; k++) {
                    System.out.printf("%d : %d : %d%n", i, j, k);
                }
            }
        }
    }
}
